W-11722162-email-verification (#235)
mulesoft/docs-access-management: modules/ROOT/pages/_partials/include-nav-steps-to-profile.adoc

. Log in to Anypoint Platform.

> . In the Anypoint Platform navigation bar, click the circle icon that contains your organization or business group's acronym.

Reviewer: The circle icon contains the initials of the user, not the org or BG acronym. Am I missing anything?

Author: ```suggestion
. In the Anypoint Platform navigation bar, click the circle icon that contains the initials associated with your Anypoint Platform account.
```
